skyportal/facility_apis/observation_plan.py

```diff
@@ -22,16 +23,23 @@
 from baselayer.app.flow import Flow
 from baselayer.app.env import load_env
 
+from ..email_utils import send_email
 from . import FollowUpAPI
 
 log = make_log('api/observation_plan')
 
 env, cfg = load_env()
 
+SLACK_URL = f"{cfg['slack.expected_url_preamble']}/services"
+
 default_filters = cfg['app.observation_plan.default_filters']
 
 use_skyportal_fields = cfg['app.observation_plan.use_skyportal_fields']
 
+email = False
+if cfg["email_service"] == "sendgrid" or cfg["email_service"] == "smtp":
```

Comment:
Should these be a `get` in case `email_service` is not defined in the config?

Reply:
Fixed